crates/nu-utils/standard_library/std.nu

```diff
@@ -8,20 +8,31 @@ def _assert [
 }
 
 # ```nushell
+# >_ let a = 3
 # >_ assert ($a == 3)
 # >_ assert ($a != 3)
 # Error:
-#   × condition given to `assert` does not hold
-#    ╭─[entry #12:5:1]
-#  5 │     if not $cond {
-#  6 │         error make {msg: $msg}
-#    ·         ─────┬────
-#    ·              ╰── originates from here
-#  7 │     }
+#   × Assertion failed: No message
+#    ╭─[nushell/crates/nu-utils/standard_library/std.nu:6:1]
+#  5 │ let a = 3
+#  6 │ assert ($a == "3")
+#    ·         ────┬────
+#    ·             ╰── It is not true.
+#  7 │ assert ($a != 3)
 #    ╰────
 # ```
-export def assert [cond: bool] {
-    _assert $cond "condition given to `assert` does not hold"
+export def assert [cond: bool, message?: string] {
+    if $cond { return }
+    let msg_text = (if ($message != null) { $message } else { "No message" })
```

Comment:
you can use
```rust
$message | default "No message"
```
here :+1: 

i think it's a bit cleaner maybe?

Reply:
Good idea, thank you.

Reply:
I personally dislike the "No message" message. It doesn't really provide guidance and more confusing noise.

Reply:
@sholderbach I agree, removed that.